### PR TITLE
Added urls and extraction code

### PR DESCRIPTION
**Pull request by MukulSingh105**

### CODE REVIEW

File: ConditionalFormatting/scripts/cf-rule-extraction/Dockerfile.linux [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# (Cache dotnet restore, so we only re-fetch everything from the internet when the .csproj changes.)
+COPY . /src
+RUN cd /src && dotnet restore -r linux-x64
+RUN cd /src && dotnet publish --no-restore -c Release -r linux-x64 --self-contained true
+
+# Multi-stage build, to keep the final output clean.
+FROM ${BASE_IMAGE}
+COPY --from=build /src/bin/Release/net5.0/linux-x64/publish /xlsx-to-json
+ENV PATH=/xlsx-to-json:${PATH} PIPELINE_MAP_COMMAND="/xlsx-to-json/xlsx-to-json -i {input} -o {outputDir}" PIPELINE_MAP_SERVER_COMMAND="/xlsx-to-json/xlsx-to-json --server {endpoint}"
+#ENV PATH=/xlsx-to-json:${PATH} PIPELINE_MAP_COMMAND="/xlsx-to-json/xlsx-to-json --inputFileId {inputFileId} -i {input} -o {outputDir}"
```
Comment (josepablocam) [inline ConditionalFormatting/scripts/cf-rule-extraction/Dockerfile.linux:16]: Can we remove this if commented out?

File: ConditionalFormatting/scripts/cf-rule-extraction/Dockerfile.linux [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Build with: pipeline-tool docker-build --image-name excel-online-calc-test/commonimages/1-xlsx-to-json [--build-loc <this directory>]
```
Comment (josepablocam) [inline ConditionalFormatting/scripts/cf-rule-extraction/Dockerfile.linux:1]: What is pipeline-tool? Is this perhaps a stale comment specifically for the internal setup?

**Review by josepablocam — APPROVED** [review]
Two minor comments but otherwise good for me.

File: ConditionalFormatting/scripts/cf-rule-extraction/Dockerfile.linux (hunk 2 above)
Comment (MukulSingh105) [reply ConditionalFormatting/scripts/cf-rule-extraction/Dockerfile.linux]: Yup. Removed.

**Pull request merged**